portal/models/fhir.py

```diff
@@ -526,11 +567,31 @@ def consolidate_answer_pairs(answers):
 
         return filtered_answers
 
+    def entry_method(row_data, qnr_data):
+        # Todo: replace with EC.PAPER CodeableConcept
```

Comment:
I was referring to replacing the string `'paper'` with the coded value `EC.PAPER.code` (or however the correct syntax is)